.github/workflows/libc-fullbuild-tests.yml

```diff
@@ -11,12 +11,18 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-24.04
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         include:
-          - c_compiler: clang
+          - os: ubuntu-24.04
+            ccache-variant: sccache
+            c_compiler: clang
+            cpp_compiler: clang++
+          - os: ubuntu-24.04-arm
+            ccache-variant: ccache
+            c_compiler: clang
```

Comment:
Are you sure that these two will invoke clang correctly? In the past, I had to use `CC="ccache clang"` to get ccache to fall back to clang upon cache miss.

Reply:
This is effectively what `CMAKE_{C,CXX}_COMPILER_LAUNCHER` do.